styles/styles.css

```diff
@@ -185,7 +200,19 @@ main img {
 
 /* sections */
 main .section {
-  padding: 64px 16px;
+  padding: 64px 0;
+}
+
+@media (width >= 600px) {
+  main .section {
+    padding: 64px 0;
```

Comment:
Why is this needed?

Reply:
I think I changed it because it got in the way of the margin the media gallery needed to have. I could just use this for the space needed, if you think it is better this way, but it needs to be adjusted regardless

Reply:
I mean that it looks like it should not be needed because it has the same exact value as the default right above